### ISSUE TITLE
Pristine-1.3: Data transfer constants must be updated 

### ISSUE THREAD

**Issue by kewinrausch** [IRATI/stack#790]
Actually are missing some data transfer constant which are relative to the Rate Based Flow Control.
A trace of this can be found at the begin of serdes.c.

```
/* FIXME: To be also defined in dt_cons when rate based fc is added */
#define RATE_LEN      4
#define TIME_LEN      4
```

**Comment by kewinrausch** [comment]
@lbergesio contribution seems to have resolved this in my pristine-1.3-rbfc branch. 
Once merged with the pristine-1.3 official branch then this issue can be closed.

**Comment by kewinrausch** [comment]
Branch merged, problem solved.
